src/TurbulenceClosures/turbulence_closure_implementations/isopycnal_skew_symmetric_diffusivity.jl

```diff
@@ -60,7 +60,9 @@ IsopycnalSkewSymmetricDiffusivity(FT::DataType; kw...) =
     IsopycnalSkewSymmetricDiffusivity(VerticallyImplicitTimeDiscretization(), FT; kw...)
 
 function with_tracers(tracers, closure::ISSD{TD, A, N}) where {TD, N}
-    κ_skew = !isa(closure.κ_skew, NamedTuple) ? closure.κ_skew : tracer_diffusivities(tracers, closure.κ_skew)
+    # For the moment, allow only one skew coefficient for all tracers
+    # TODO: maybe generalize it?
+    κ_skew = closure.κ_skew 
     κ_symmetric = !isa(closure.κ_symmetric, NamedTuple) ? closure.κ_symmetric : tracer_diffusivities(tracers, closure.κ_symmetric)
```

Comment:
doesn't `tracer_diffusivities` already handle the `NamedTuple` case?

Reply:
Ah I have already changed it 😅

Reply:
word